packages/io/stl-serializer/tests/ascii.test.js

```diff
@@ -1,39 +1,19 @@
 const test = require('ava')
-const { CSG } = require('@jscad/csg')
-const serializer = require('./index.js')
 
-test('serialize csg to stl (binary)', function (t) {
-  const csg1 = new CSG.cube()
-  const observed = serializer.serialize({ binary: true }, csg1)
+const { primitives, transforms } = require('@jscad/modeling')
```

Comment:
where did the binary test go ? :)

Reply:
aah never mind, saw the other file

Reply:
why split them though ?

Reply:
The both ascii and binary test need improvement, which should increase the number of test cases, and the lines of code. So, I just split now in preparation.

Reply:
not sure I follow ? why do we need more tests ?

Reply:
As the note says in the binary test...

// TODO: VERY shallow testing ... improve

Reply:
fair point